### PR TITLE
Add script for generating bigpoly test-data

### PR DESCRIPTION
**Pull request by jonathan-eq**
**Approach**
✏️ 

(Screenshot of new behavior in GUI if applicable)

- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,6 @@ src/ert/shared/version.py
 /compile_commands.json
 *_pb2.py
 *_pb2.pyi
+/test-data/bigpoly/*
+!/test-data/bigpoly/make_bigpoly.sh
+!/test-data/bigpoly/generate_eclsum.py
```
Comment (berland) [inline .gitignore:39]: do we need these lines here?

Comment (jonathan-eq) [reply .gitignore:39]: I think we should add them for normal poly_example too. It is very easy to commit logs and poly_out by mistake

**Comment by berland** [comment]
Should we have a PR in gkomodo-releases that demonstrates that it is able to build on this PR?

It should get the grdecl script from Ert and generate the remaining queue system tests based on the files dumped by this script.

**Comment by xjules** [comment]
> Should we have a PR in gkomodo-releases that demonstrates that it is able to build on this PR?
> 
> It should get the grdecl script from Ert and generate the remaining queue system tests based on the files dumped by this script.

Yes, I wanted to suggest the same. Also to have a small description how to run / build big poly or is it so simple to run the `make_bigpoly` should be enough?

**Comment by jonathan-eq** [comment]
> > Should we have a PR in gkomodo-releases that demonstrates that it is able to build on this PR?
> > It should get the grdecl script from Ert and generate the remaining queue system tests based on the files dumped by this script.
> 
> Yes, I wanted to suggest the same. Also to have a small description how to run / build big poly or is it so simple to run the `make_bigpoly` should be enough?

I want to add a file `modify_bigpoly.sh` in komodo-releases/
The script will checkout the bigpoly script from ert, run it, then append the other queue_options for cluster

**Comment by xjules** [comment]
After giving `big-poly` some thoughts I've realized that this case / test should remain only in komodo IMHO. Semantically it only serves as a testing detail related to our production environment and as such it does not provide any value to potential users of ert. Any thoughts @sondreso @berland

**Comment by jonathan-eq** [comment]
> After giving `big-poly` some thoughts I've realized that this case / test should remain only in komodo IMHO. Semantically it only serves as a testing detail related to our production environment and as such it does not provide any value to potential users of ert. Any thoughts @sondreso @berland

When refactoring snapshots, it was very useful to have a heavy run, so we had to manually clone over bigpoly from komodo-releases. This was more difficult than it had to be, and most of the bigpoly files created by the script are not for running on cluster anyways. 

Instead of manually cloning the script from komodo-releases and removing the cluster side of it, i think it is better to have the essential part of the script in Ert, and a smaller script for bigpoly_cluster in komodo.

**Comment by xjules** [comment]
Suggestion: Let's create a new issue, which would aim to have such a case in `test-data`, where we should give it a better name (than big-poly) and make a readme, which describes utilization of such a case; ie. stress-testing the messaging system and internationalization.

**Comment by berland** [comment]
It probably belongs in `tests/performance_tests`

**Comment by xjules** [comment]
> It probably belongs in `tests/performance_tests` as a `pytest` test.

do you mean to move the entire big-poly there?

**Comment by berland** [comment]
> > It probably belongs in `tests/performance_tests` as a `pytest` test.
> do you mean to move the entire big-poly there?

Probably a more carefully written test setup that in some way stresses the message system without touching implementation details, with and without some mocked cpu-intensive internalization.

**Comment by jonathan-eq** [comment]
> It probably belongs in `tests/performance_tests` as a `pytest` test.

I don't think so. I mainly use bigpoly for manually testing the GUI. If we want a pytest performance test for it, we can create a fixture and use it the same way we use `copy_poly_example`. It is easier to do it this way than the other way around.

**Pull request closed without merging**